<launch>
    <!-- vim: set ft=xml noet : -->
    <!-- <remap from="mavros/vision_pose/pose" to="vo_pose" /> -->
    <!-- <arg name="fcu_url" default="/dev/serial0:921600" /> -->
    <arg name="fcu_url" default="/dev/ttyTHS2:115200" />
    <!-- <arg name="gcs_url" default="udp://10.0.1.46:9000@10.0.0.206:6000?ids=1,255,252"/> -->
    <arg name="gcs_url" default=""/>
    <arg name="tgt_system" default="1" />
    <arg name="tgt_component" default="1" />
    <arg name="log_output" default="screen" />
    <arg name="fcu_protocol" default="v2.0" />
    <arg name="respawn_mavros" default="true" />

    <include file="$(find mavros)/launch/node.launch">
        <arg name="pluginlists_yaml" value="$(find mavros)/launch/apm_custom_pluginlists.yaml" />
        <arg name="config_yaml" value="$(find mavros)/launch/apm_custom_config.yaml" />
        <arg name="fcu_url" value="$(arg fcu_url)" />
        <arg name="gcs_url" value="$(arg gcs_url)" />
        <arg name="tgt_system" value="$(arg tgt_system)" />
        <arg name="tgt_component" value="$(arg tgt_component)" />
        <arg name="log_output" value="$(arg log_output)" />
        <arg name="fcu_protocol" value="$(arg fcu_protocol)" />
        <arg name="respawn_mavros" default="$(arg respawn_mavros)" />
    </include>
</launch>
